packages/cactus-api-client/src/main/typescript/socketio-api-client.ts

```diff
@@ -0,0 +1,345 @@
+/*
+ * Copyright 2020-2021 Hyperledger Cactus Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * SocketIOApiClient.ts
+ */
+
+// TODO - Document it
+
+const defaultMaxCounterRequestID = 100;
+const defaultSyncFunctionTimeoutMillisecond = 5000;
+
+import { Logger, Checks } from "@hyperledger/cactus-common";
+import { LogLevelDesc, LoggerProvider } from "@hyperledger/cactus-common";
+import { ISocketApiClient } from "@hyperledger/cactus-core-api";
+
+import { Socket, SocketOptions, ManagerOptions, io } from "socket.io-client";
+import { readFileSync } from "fs";
+import { resolve as resolvePath } from "path";
+import { verify, VerifyOptions } from "jsonwebtoken";
+import { Observable, ReplaySubject } from "rxjs";
+import { finalize } from "rxjs/operators";
+
+// TODO - Remove any from here
+export function verifyValidatorJwt(
+  keyPath: string,
+  targetData: string,
+): Promise<any> {
+  return new Promise((resolve, reject) => {
+    const publicKey = readFileSync(resolvePath(__dirname, keyPath));
```

Comment:
@outSH Could this function benefit from being made asynchronous instead of blocking?

Reply:
I've changed this to `readFile()`, and also fixed fixed all `any` types in this function.